api/app/utils.py

```diff
@@ -0,0 +1,19 @@
+import random
+import string
+import httpx
+from .config import get_settings
+
+
+def generate_token():
+    return "".join(
+        random.choice(string.ascii_uppercase + string.ascii_lowercase + string.digits)
+        for _ in range(4)
+    )
+
+
+async def fatsms_send_sms(message: str, to_phone: str):
+    url = "https://fatsms.com/send-sms"
```

Comment:
Would prob make sense to also put in config.py, but not .env

Reply:
Done :)